test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -691,6 +691,13 @@ protected String createType(String keyspace, String query)
         return typeName;
     }
 
+    protected String createTypeName()
```

Comment:
`createType` should use `createTypeName`, same as in 4.1 and trunk.

Reply:
Done